packages/client/src/rtc/codecs.ts

```diff
@@ -21,9 +21,9 @@ export const getPreferredCodecs = (
     const codec = c.mimeType.toLowerCase();
     logger?.('debug', `Found supported codec: ${codec}`);
     const shouldRemoveCodec =
-      codecToRemove && codec === `${kind}/${codecToRemove}`;
+      codecToRemove && codec === `${kind}/${codecToRemove.toLowerCase()}`;
```

Comment:
this is an unrelated fix, without this if case was upper case for codec passed, it did not work